content/en/monitors/monitor_types/log.md

```diff
@@ -10,88 +10,68 @@ further_reading:
 - link: "monitors/monitor_status"
   tag: "Documentation"
-  text: "Consult your monitor status"
+  text: "Check your monitor status"
 ---
 
-{{< img src="monitors/monitor_types/log/log_monitor_overview.png" alt="Log monitor overview" responsive="true" >}}
-
 ## Overview
 
-Log monitors alert when a specified type of log exceeds a user-defined threshold over a given period of time.
-
-## Setup
-
-Construct a query to control what is monitored:
-
-1. Define the search query:
-    {{< img src="monitors/monitor_types/log/define_the_search_query.png" alt="Define the search query" responsive="true" style="width:50%;" >}}
-    The search query has the same behavior as [the log explorer search][1]
-    
-2. Choose a [Measure][1] or [Facet][2] to monitor. [Measure][1] lets you choose the aggregation function whereas [Facet][2] displays a count.
-    
-    {{< img src="monitors/monitor_types/log/choose_measure_facet.png" alt="choose measure facet" responsive="true" style="width:50%;">}}
-    
-3. Select the aggregation function for the [Measure][1] you want to monitor:
-    
-    {{< img src="monitors/monitor_types/log/agg_function.png" alt="aggregation function for Log Analytics" responsive="true" style="width:50%;">}}
-
-4. (Optional) Define the alert grouping:
-  {{< img src="monitors/monitor_types/log/log_monitor_group_by.png" alt="Set alert conditions" responsive="true" style="width:50%;" >}}
-    With or without alert grouping defined, you get **one** alert when the aggregated value meets the conditions set below. Even if you split the query by host, a single notification is sent if several hosts meet the conditions set below. This is done to reduce notification noise.
-
-5. Set alert conditions. The following options can be used:
-
-  {{< img src="monitors/monitor_types/log/above_below.png" alt="aggregation function for Log Analytics" responsive="true" style="width:50%;">}}
+A logs monitor alerts when a specified type of log exceeds a user-defined threshold over a given period of time.
 
-6. Configure your **notification options**:  
+## Monitor creation
 
-  {{< img src="monitors/monitor_types/log/set_alert_conditions.png" alt="Set alert conditions" responsive="true" style="width:50%;" >}}
+To create a [logs monitor][1] in Datadog, use the main navigation: *Monitors --> New Monitor --> Logs*.
 
-  Refer to the [notifications][2] dedicated documentation page for detailed options.
-    
+### Define the search query
 
-## Notifications and log samples
+As you define the search query, the graph above the search fields updates.
 
-It is possible to add up to 10 samples of logs that triggered the monitor in the notification message.
-This is available for Slack, Jira, Webhook, Microsoft Teams, and email notifications.
+* If you have [multiple log indexes][2], select the index to search.
+* Construct a search query using the same logic as a [log explorer search][3].
+* Choose a [facet][4] or [measure][5] to monitor.
```

Comment:
There are two kinds of Log monitor:

* Monitor over a log count (where no measures/facets are selected.. only a search (optional) in the search bar), which means that dd evaluates the amount of logs over a selected time frame then compare it to the threshold conditions

* Monitor over a facet/measure: If a facet is selected then the monitors alerts over the `Unique value count` of this facet. If a measure is selected then it's kinda like a metric monitor, and aggregation needs to be selected as well.

Reply:
+100